e2e/test/scenarios/metrics/browse.cy.spec.ts

```diff
@@ -230,4 +231,60 @@ describe("scenarios > browse > metrics", () => {
       getMetricsTableItem(3).should("contain", "Metric D");
     });
   });
+
+  describe("dot menu", () => {
+    it("should be possible to bookmark a metrics from the dot menu", () => {
```

Comment:
Could you check here the metric appears and disappears from the bookmark list in the sidebar?